### PR TITLE
Enhancement: Enable `braces_position` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables the `braces_position` fixer
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/v3.40.2/doc/rules/basic/braces_position.rst.

### CODE REVIEW

File: images/logo.php [hunk 1]
```diff
@@ -19,7 +19,8 @@
     }
 }
 
-function imgheader($filename) {
+function imgheader($filename)
+{
```
Comment (kamil-tekiela) [inline images/logo.php:23]: I don't like this. Also, I think this fixer should be applied together with indentation fixer.

Comment (localheinz) [reply images/logo.php:23]: Maybe #886 helps, @kamil-tekiela?

Comment (kamil-tekiela) [reply images/logo.php:23]: To be honest, I would hold off with applying either of them. It creates too many changes and I don't know if this project is ready for this yet. I'd rather take small steps here.

**Comment by localheinz** [comment]
I could rebase this pull request after the merge of #934.

**Comment by localheinz** [comment]
Closing as requested by @pronskiy.

**Pull request closed without merging**